modules/hcp-custom-ovn-subnets.adoc

```diff
@@ -0,0 +1,187 @@
+        "clusterNetworkOperator": {
+          "ovnKubernetesConfig": {
+            "ipv4": {
+              "internalJoinSubnet": "100.99.0.0/16",
+              "internalTransitSwitchSubnet": "100.69.0.0/16"
+            }
+          }
+        }
+      }
+    }
+  }'
+----
++
+where:
+
+`<hosted_cluster_name>`:: Specifies the name of the hosted cluster.
+`<hosted_control_plane_namespace>`:: Specifies the name of the hosted control namespace.
+`ipv4`:: Both subnet fields in this section must be in a valid IPv4 CIDR notation, such as `192.168.1.0/24`. The prefix range is `/0` to `/30`, inclusive. The first octet cannot be 0, and the string length must be 9-18 characters. The subnet fields cannot use the same value. The subnet must be large enough to accommodate one IP address per node in the cluster. When you plan subnet size, consider future cluster growth. If you omit these fields, the default value for the `internalJoinSubnet` field is `100.64.0.0/16`, and the default value for the `internalTransitSwitchSubnet` field is `100.88.0.0/16`.
+
+.Verification
+
+. Verify that the configuration is correct by entering the following command:
++
+[source,terminal]
+----
+$ oc get hostedcluster <hosted_cluster_name> -n <hosted_control_plane_namespace> \
+  -o jsonpath='{.spec.operatorConfiguration.clusterNetworkOperator.ovnKubernetesConfig}' | jq .
+----
++
+.Example output
+[source,terminal]
+----
+{
+  "ipv4": {
+    "internalJoinSubnet": "100.99.0.0/16",
+    "internalTransitSwitchSubnet": "100.69.0.0/16"
+  }
+}
+----
+
+. Check the Network Operator configuration in the hosted cluster by entering the following commands:
++
+[source,terminal]
+----
+$ oc extract secret/<hosted_cluster_name>-admin-kubeconfig -n <hosted_control_plane_namespace> --to=- > hosted-kubeconfig
+----
++
+[source,terminal]
+----
+$ oc get network.operator.openshift.io cluster \
+  --kubeconfig=hosted-kubeconfig \
+  -o jsonpath='{.spec.defaultNetwork.ovnKubernetesConfig.ipv4}' | jq .
+----
++
+.Example output
++
+[source,terminal]
+----
+{
+  "internalJoinSubnet": "100.99.0.0/16",
+  "internalTransitSwitchSubnet": "100.69.0.0/16"
+}
+----
+
+. Verify that the OVN pods are running by entering the following commands:
++
+[source,terminal]
+----
+$ oc rollout status daemonset/ovnkube-node \
+  -n openshift-ovn-kubernetes \
+  --kubeconfig=hosted-kubeconfig \
+  --timeout=5m
+----
++
+[source,terminal]
+----
+$ oc get pods -n openshift-ovn-kubernetes --kubeconfig=hosted-kubeconfig
+----
++
+All `ovnkube-node` pods should be in `Running` state with all containers ready (8/8).
+
+. Test network connectivity by entering the following commands:
++
+[source,terminal]
+----
+$ oc run test-pod-1 --image=registry.access.redhat.com/ubi8/ubi-minimal \
+  --command -- sleep infinity --kubeconfig=hosted-kubeconfig
+----
++
+[source,terminal]
+----
+$ oc run test-pod-2 --image=registry.access.redhat.com/ubi8/ubi-minimal \
+  --command -- sleep infinity --kubeconfig=hosted-kubeconfig
```

Comment:
If execute the command directly, i think will meet the issue: pods "test-pod-2" is forbidden: violates PodSecurity "restricted:latest": allowPrivilegeEscalation != false (container "test-pod-2" must set securityContext.allowPrivilegeEscalation=false), you can refer the case:[OCP-83958](https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitem?id=OCP-83958) to write about  network connectivity part.

Reply:
Thanks, @wewang58. Unfortunately, I'm having issues with the VPN today, so I can't view that test case. Could you provide an example of what command might work instead of `$ oc run test-pod-2 --image=registry.access.redhat.com/ubi8/ubi-minimal \
  --command -- sleep infinity --kubeconfig=hosted-kubeconfig`?

Reply:
@wewang58 I was finally able to solve my VPN issues and view the test case. I updated the verification steps based on the test case. PTAL and let me know if anything should change. Thank you!

Reply:
@lahinson You can use the command: oc run test-pod-2 --image=registry.access.redhat.com/ubi8/ubi-minimal --overrides='{"spec":{"securityContext":{"runAsNonRoot":true,"seccompProfile":{"type":"RuntimeDefault"}}}}'  -- sleep infinity